canonical-web-and-design/ubuntu.com
# Move /core/smartstart to /core/services

## Done

- Moved the page at /core/smartstart to /core/services and updated it to match the [copy doc](https://docs.google.com/document/d/11z973-P-vuaVp7gCQpoQly0gzqNMYX7ETPVghEkDDsE/edit#)
- Added redirects
- Also updated minor changes to smart start copy and links on:
  - /core
  - /internet-of-things/smart-displays
  - /industrial

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-11508.demos.haus/core/services
    - Be sure to test on mobile, tablet and desktop screen sizes
- See that the page matches the copy doc
- Visit:
  - https://ubuntu-com-11508.demos.haus/core, see that it matches the [copy doc](https://docs.google.com/document/d/1QoM0Yqx4JfDJ2G3OPt5bds-ximG1jASmU_PWX4iff3E/edit#heading=h.qlnm9gmavgl5)
  - https://ubuntu-com-11508.demos.haus/internet-of-things/smart-displays, see that it matches the [copy doc](https://docs.google.com/document/d/1fFvBl-ZZvKDFV6OMv7O4kudQKtyxfvSU60rfVdKluAQ/edit#)
  - https://ubuntu-com-11508.demos.haus/industrial, see that it matches the [copy doc](https://docs.google.com/document/d/1QiU75g49lRKebt82oyLSxR3-PU96Sk87nqauAZ4vvIY/edit#)

## Issue / Card

Fixes:
- https://github.com/canonical-web-and-design/web-squad/issues/5080
- https://github.com/canonical-web-and-design/web-squad/issues/5083
- https://github.com/canonical-web-and-design/web-squad/issues/5084
- https://github.com/canonical-web-and-design/web-squad/issues/5085
